File: AppMaker/app/src/main/java/com/example/appmaker/SecondActivity.java
package com.example.appmaker;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {

    TextView txtView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        txtView = findViewById(R.id.txtView);

        Intent fromMain = getIntent();

        String userName = fromMain.getStringExtra("keyYourName");
        String appName = fromMain.getStringExtra("keyAppName");

        txtView.setText(appName +  " is developed by " + userName);



    }
}

File: BundlePassingKotlin/app/src/main/java/com/himanshu/bundlepassingkotlin/SecondActivity.kt
package com.himanshu.bundlepassingkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import android.widget.TextView

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        val textView: TextView = findViewById(R.id.textView)

        val displayIncomingMsg = intent.getStringExtra("userMsg")


        textView.text = displayIncomingMsg






    }
}